packages/manager/modules/catalog-price/src/catalog-price.component.js

```diff
@@ -9,6 +9,8 @@ const component = {
     price: '<',
     tax: '<',
     user: '<',
+    cdnCase: '<',
```

Comment:
I'm a bit afraid to see that we introduce a specific product case in this common `catalog-price` module. Do we really need to do it here ?
Could we have something more generic ?

Reply:
yes it could be great to have somthing more generic but iam not yet comfortable to make thoses changes.
Can we do it togther please ?